src/Microsoft.ML.ImageAnalytics/VectorToImageTransform.cs

```diff
@@ -336,7 +336,7 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
 
                 var type = _types[iinfo];
                 var ex = _parent._columns[iinfo];
-                bool needScale = ex.Offset != 0 || ex.Scale != 1;
+                bool needScale = ex.OffsetPixelColor != 0 || ex.ScalePixelColor != 1;
```

Comment:
>OffsetPixelColor [](start = 36, length = 16)

is this correct? the offset are for the image, not for the pixel color? (one can argue that the pixel colors are the image, but i think it becomes too complicated. )

Reply:
Agree. Changing to `ScaleImage` and `OffsetImage`.